app/controllers/tag_lists_controller.rb

```diff
@@ -45,7 +39,30 @@ def group
     @group ||= Group.find(params[:group_id])
   end
 
+  def manageable_people
+    @manageable_people ||= people.select { |person| current_ability.can?(:manage_tags, person) }
+  end
+
+  def people
+    @people ||= Person.includes(:tags).where(id: people_ids).uniq
+  end
+
+  def people_ids
+    return [] if params[:ids].nil?
+    params[:ids].to_s.split(',')
+  end
+
+  def tags
+    @tags ||= ActsAsTaggableOn::Tag.find_or_create_all_with_like_by_name(tag_names)
+  end
+
+  def tag_names
+    return [] if params[:tags].nil?
+    return params[:tags].keys if params[:tags].is_a?(Hash)
```

Comment:
Können die tagliste tatsächlich als hash übertragen werden? Falls nein, dann löschen

Reply:
Ja, die von Checkboxes generierten params sind in der Form { tagname: 'on', ... }